src/common/page-queries/transactions.ts

```diff
@@ -5,16 +5,43 @@ import { DEFAULT_LIST_LIMIT } from '@common/constants';
 import { DEFAULT_TX_FILTER_TYPES } from '@store/recoil/filter';
 
 export const getTransactionsPageQueries: GetQueries = async ctx => {
-  const { transactionsApi } = await getApiClients(ctx);
+  const { bnsApi, transactionsApi } = await getApiClients(ctx);
   return [
     [
       getTxQueryKey.confirmed(DEFAULT_LIST_LIMIT, DEFAULT_TX_FILTER_TYPES),
       async () => {
-        return (await transactionsApi.getTransactionList({
+        let transactions = (await transactionsApi.getTransactionList({
           limit: DEFAULT_LIST_LIMIT,
           offset: 0,
           type: DEFAULT_TX_FILTER_TYPES,
         })) as TransactionsListResponse;
+
+        for (let i = 0; i < transactions.results.length; i++) {
+          let res;
+
+          res = await bnsApi.getNamesOwnedByAddress({
+            address: transactions.results[i].sender_address,
+            blockchain: 'stacks',
+          });
+          // @ts-ignore
```

Comment:
Where possible, we try to avoid ts-ignore checks. 

Here, the type errors are because `transactions.results[i]` is type Transaction, but `sender_address` only exists on `TokenTransferTransaction`. We can satisfy the compiler by checking the tx has `tx_type === 'token_transfer'`